repository/h.json

```diff
@@ -1538,6 +1538,19 @@
 				}
 			]
 		},
+		{
+			"name": "Hyperscript",
+			"details": "https://github.com/gnat/hyperscript-sublime",
+			"labels": ["language syntax", "syntax", "html", "script", "highlighting", "htmx"],
+			"author": "gnat",
+			"description": "Syntax highlighting for Hyperscript (companion of HTMX)",
```

Comment:
It's easier to set the description on your GitHub repo so you can manage it without having to update your entry here.

Also, you don't need to set "author" if it's just your username, package control will pick that up from the repo.